static/app/utils/profiling/hooks/useResizableDrawer.tsx

```diff
@@ -57,8 +57,13 @@ export function useResizableDrawer(options: UseResizableDrawerOptions): {
       const isXAxis = options.direction === 'left' || options.direction === 'right';
       const isInverted = options.direction === 'down' || options.direction === 'left';
 
+      document.body.style.pointerEvents = 'none';
       document.body.style.userSelect = 'none';
 
+      // We've disabled pointerEvents on the body, the cursor needs to be
+      // applied to the root most element to work
+      document.documentElement.style.cursor = isXAxis ? 'ew-resize' : 'ns-resize';
```

Comment:
Should we remember if any non default cursor was set before we took control so we can reset to the same value? Doesnt seem like it could happen too often tbh but it may save us some time with debugging